### PR TITLE
Add CLI params for disabling deleting and config/policy/cors updates

### PR DESCRIPTION
**Pull request by orm-forks**
We have added a bit more flexibility in what the plugin does to the S3 bucket as part of the flow.

### CODE REVIEW

**Comment by amsross** [comment]
I personally love this. Could you add some more information to the `README`, though? Also, spacing issues.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ echo "error page" >> client/dist/error.html
 **Fourth**, run the plugin, and visit your new website!
 
 ```
-serverless client deploy [--stage $STAGE] [--region $REGION]
+serverless client deploy [--stage $STAGE] [--region $REGION] [--no-delete-contents] [--no-config-change] [--no-policy-change] [--no-cors-change]
```
Comment (fernando-mc) [inline README.md]: Does the `no` in all the flags get removed somewhere?

These options make sense to me. Just have to figure out how to refactor some of the documentation to explain these options. And also decide if these make sense as command line args or as config details in serverless.yml.

File: index.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Client {
     this.serverless = serverless;
     this.provider = 'aws';
     this.aws = this.serverless.getProvider(this.provider);
+	this.options = options;
```
Comment (fernando-mc) [inline index.js]: Can you remove the tabs and add in spaces everywhere? You can review in GitHub to compare the spacing here.

**Comment by linusmarco** [comment]
@orm-forks, if you have any questions or need any help merging the recent linting/docs changes into this PR, just let me know.

**Comment by tim-stasse** [comment]
+1

This would be excellent to have!

Can someone else please fix this if the author doesn't do it soon?

**Comment by linusmarco** [comment]
I'm working on a major rewrite of the library that should be done soon. I can probably incorporate this into that PR

**Comment by linusmarco** [comment]
@tim-stasse This is now live in v2.0.0!

**Pull request closed without merging**